CHANGELOG.md

```diff
@@ -33,7 +33,7 @@
 * [BUGFIX] Rules: improve error handling when querier is local to the ruler. #7567
 * [BUGFIX] Querier, store-gateway: Protect against panics raised during snappy encoding. #7520
 * [BUGFIX] Ingester: Prevent timely compaction of empty blocks. #7624
-* [BUGFIX] querier: Don't cache context.Canceled errors for bucket index. #7620
+* [BUGFIX] Querier: Don't cache context.Canceled errors for bucket index. #7620
```

Comment:
I think this is a typo all the others are capitalised.

Reply:
It's OK editing it, but we're not very strict with the CHANGELOG. Anyway, it's good you improved it.